### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Mihndim2020**
For this pull request, I have; 
- Initialize React app,
- Add React Redux,
- Structure files using ducks pattern,
- Added React Router,
- Display a list of books with add/remove functionality,
- Display category page is under construction.

### CODE REVIEW

File: build/static/js/2.cacd5bae.chunk.js.LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+ */
+
+/** @license React v17.0.2
+ * react.production.min.js
+ *
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
```
Comment (Emmyn5600) [inline build/static/js/2.cacd5bae.chunk.js.LICENSE.txt]: - [x] Kindly, remove the unnecessary files in both build and public folders, there are duplicate files that are not necessary in our project.
react6

File: src/App.js [hunk 2]
```diff
@@ -1,58 +1,57 @@
...
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import NavBar from './features/components/NavBar';
+import Book from './features/components/Book';
+import AddBook from './features/components/AddBook';
+import BookCategories from './features/components/BookCategories';
```
Comment (Emmyn5600) [inline src/App.js:10]: - [x] Routes has been implemented well. I can see the categories and booklist

File: src/features/components/AddBook.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        <option value="action">Action</option>
+        <option value="science-fiction">Science Fiction</option>
+        <option value="economy">Economy</option>
+      </select>
+      <input className="add-book-button" type="submit" value="ADD BOOK" />
+    </div>
+  </>
+);
+
+export default AddBook;
```
Comment (Emmyn5600) [inline src/features/components/AddBook.js:19]: - [x] The structure of the files should follow the ducks pattern.

---------------------------------------------Yours------------------------------------------

react7

---------------------------------------------Original/Requirement------------------------------------------

react8

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Mihndim2020 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
   - The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- Categories still under construction:heavy_check_mark:
- Booklist renders the list of books:heavy_check_mark:

Changes required :writing_hand:  

      * #### Please check the comments under the review.
       
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/features/components/AddBook.js (hunk 3 above)
Comment (Mihndim2020) [reply src/features/components/AddBook.js:19]: File structure updated as requested

File: build/static/js/2.cacd5bae.chunk.js.LICENSE.txt (hunk 1 above)
Comment (Mihndim2020) [reply build/static/js/2.cacd5bae.chunk.js.LICENSE.txt]: Build folder and unnecessary files deleted

**Comment by Mihndim2020** [comment]
Requested changes effected. Thank you for your feedback.

File: build/static/js/2.cacd5bae.chunk.js.LICENSE.txt (hunk 1 above)
Comment (Emmyn5600) [reply build/static/js/2.cacd5bae.chunk.js.LICENSE.txt]: Well solved

File: src/features/components/AddBook.js (hunk 3 above)
Comment (Emmyn5600) [reply src/features/components/AddBook.js:19]: Well solved, however you are missing configure.js file

**Review by Emmyn5600 — APPROVED** [review]
Hi, 

Good job so far! 👏🏼 @Mihndim2020  

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE
 
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**